### PR TITLE
Bert make PR

### PR DESCRIPTION
**Pull request by bert-griffin**

### CODE REVIEW

**Comment by wanda-carlson** [comment]
This looks cool

**Comment by bert-griffin** [comment]
How about now ?

**Comment by wanda-carlson** [comment]
@bert-griffin Is this oaky ?

**Comment by bert-griffin** [comment]
@wanda-carlson Yes

File: src/foo.py [hunk 1]
```diff
@@ -4,4 +4,5 @@ def hello():
     print("Hello foo")
     print("Hello foo1")
     print("Hello foo3")
+    print("Hello foo3")
```
Comment (wanda-carlson) [inline src/foo.py:7]:
```suggestion
    print("Bert make change")
```

**Review by wanda-carlson — CHANGES_REQUESTED** [review]

File: src/foo.py [hunk 2]
```diff
@@ -4,4 +4,5 @@ def hello():
     print("Hello foo")
     print("Hello foo1")
     print("Hello foo3")
+    print("Bert make change")
```
Comment (wanda-carlson) [inline src/foo.py:7]:
```suggestion
    print("Bert make nice change")
```

File: src/foo.py [hunk 3]
```diff
@@ -4,4 +4,5 @@ def hello():
     print("Hello foo")
     print("Hello foo1")
     print("Hello foo3")
+    print("Bert make nice change")
```
Comment (bert-griffin) [inline src/foo.py:7]:
```suggestion
    print("Bert make nice change wow")
```

File: src/foo.py [hunk 4]
```diff
@@ -4,4 +4,5 @@ def hello():
     print("Hello foo")
     print("Hello foo1")
     print("Hello foo3")
+    print("Bert make nice change wow")
```
Comment (bert-griffin) [inline src/foo.py:7]:
```suggestion
    print("Bert make nice change wow")
    print("Bert make nice change wow")
```

**Comment by wanda-carlson** [comment]
asdfasdf

**Pull request closed without merging**